appveyor.yml

```diff
@@ -26,5 +31,19 @@ after_build:
 
     $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "DesktopClrTestResults.xml"))
 
+on_success:
+- ps: >-
+    if ($env:APPVEYOR_REPO_TAG -eq "true")
+    {
+        nuget push .\build\Castle.Core.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.log4netIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.NLogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.SerilogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+    }
+
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
```

Comment:
Ah, I thought it might be something like that. Don't mind either way, just thought it might have had some effect in the UI but it doesn't appear to.